### ISSUE TITLE
button to open LilyPond documentation in the browser

### ISSUE THREAD

**Issue by fedelibre** [frescobaldi/frescobaldi#1147]
I suggest adding a *browser icon* in the Documentation Browser's toolbar to allow to open the LilyPond documentation in the default browser (Firefox, Chrome, etc.).

In fact browsing the LilyPond documentation within Frescobaldi is useful sometimes but can be limited.

@uliska what do you think? I've marked it as beginner issue.

**Comment by uliska** [comment]
Yes, nice idea, and I agree that this usxa good "beginner" issue.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by urielo515** [comment]
could i try and develop this?

**Comment by uliska** [comment]
Of course, great!

Do you need pointers to get started at all or would you like to look for yourself first?
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by urielo515** [comment]
nice. i'm checking the code right now, but it's really large. if you could just point me to where the documentation browser toolbar is written, i guess i can work from there.

**Comment by uliska** [comment]
Ok, I"LL have a look when I'm back at the computer.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by uliska** [comment]
The documentation browser is (AFAICT) completely within the directory `frescobaldi_app/docbrowser`.

`__init__.py` provides the integration in the Tools system, the `HelpBrowser` class is not the browser itself but the "tool panel" (inheriting `panel.Panel`). The `createWidget()` method of such a panel descendant tells you where the actual visible widget is defined, namely in `browser.Browser`. So this `Browser` class is what you'll have to start looking at.

You will have to add an action in `__init__`, and you can access that in `browser.Browser` through the `ac = dockwidget.actionCollection`.

Don't hesitate to ask more ...

**Comment by listentolist** [comment]
When I right-click inside the Documentation Browser and choose to "Open Document in New Window" it already opens the current page inside the default browser. So do we really need a toolbar button for that? Or should the toolbar button link to the start page of the documentation instead of the current page? Anyway, if no one is working on it at the moment, I'd be interested in it.
